persistence-modules/java-mongodb/pom.xml

```diff
@@ -45,9 +55,43 @@
     </dependencies>
 
     <properties>
-        <mongo.version>4.8.2</mongo.version>
+        <mongo.version>5.0.1</mongo.version>
+        <slf4j.version>2.0.13</slf4j.version>
+        <junit.jupiter.version>5.7.0</junit.jupiter.version>
         <flapdoodle.version>4.4.1</flapdoodle.version>
         <morphia.version>2.0.0</morphia.version>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M5</version>
+                <configuration>
+                    <includes>
+                        <include>**/*Test.java</include>
+                    </includes>
+                    <useModulePath>false</useModulePath>
+                    <dependenciesToScan>
+                        <dependency>org.junit.jupiter</dependency>
+                    </dependenciesToScan>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.junit.jupiter</groupId>
+                        <artifactId>junit-jupiter-engine</artifactId>
+                        <version>${junit.jupiter.version}</version>
+                        <exclusions>
+                            <exclusion>
+                                <groupId>junit</groupId>
+                                <artifactId>junit</artifactId>
+                            </exclusion>
+                        </exclusions>
+                    </dependency>
+                </dependencies>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
Remove this, we have defaults in the parent pom.xml I think

Reply:
Removed